pages/categoryPage/category.css

```diff
@@ -0,0 +1,913 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500&display=swap');
```

Comment:
can you add google fonts from head in html file?

Reply:
> can you add google fonts from head in html file?

Sure, I will cange it

Reply:
I also think would be better If you set the font family for the "body" instead of setting it for each element, but it is not critical